docs: Review and edit the docs files for the new docs website

anithamuthu2022 opened the pull request:
The following changes are made to this PR: 
1. Reviewed all the files to make sure the `tier` info is correctly placed for each file. 
2. Added `<div class="enterprise-only">` for text that had ENT content. 
3. Changed the note message format to `!!! note`. 
4. Fixed the rendering issue on `ml_tutorials.md`
5. Fixed typos and added admonitions/notes in relevant pages/sections. 
6. Deleted `billing.md` page.

huguestennier commented on docs/source/guide/active_learning.md:
```diff
@@ -10,14 +10,14 @@ meta_description: Set up an end-to-end active learning loop with Label Studio us
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class=enterprise-only>
```
```suggestion
<div class="enterprise-only">
```

huguestennier commented on docs/source/guide/install_enterprise_k8s.md:
```diff
@@ -12,9 +12,7 @@ Deploy Label Studio Enterprise on a Kubernetes Cluster using Helm 3. You can use
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
```suggestion
<div class="enterprise-only"><p>
```

huguestennier commented on docs/source/guide/labeling.md:
```diff
@@ -1,10 +1,9 @@
 order: 402
 meta_title: Label and annotate data
-meta_description: "Label and annotate data to create bounding boxes, label text spans, set up relations. Filter and sort project data for machine learning dataset creation."
+meta_description: Label and annotate data using the Label Studio UI to create bounding boxes, label text spans, set up relations, and filter and sort project data for your machine learning dataset creation.
```
The original changes was a recommendation from Justin. Should we keep it?

anithamuthu2022 replied:
yes

huguestennier commented on docs/source/guide/labeling.md:
```diff
@@ -102,9 +101,7 @@ When annotators skip a task, the task no longer appears in the labeling queue fo
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
```suggestion
<div class="enterprise-only"><p>
```

huguestennier commented on docs/source/guide/manage_data.md:
```diff
@@ -55,9 +55,12 @@ You can create tabs on the data manager to [split your dataset](#Example-Split-a
+    If you make changes to the labeling configuration, any tabs in your data manager are removed. Make sure to finish the project setup before setting up complex tabs in the Data Manager.
 
-## <i class='ent'></i> Assign annotators to tasks
+<div class="enterprise"><p>
```
```suggestion
<div class="enterprise-only"><p>
```

huguestennier commented on docs/source/guide/manage_users.md:
```diff
@@ -10,9 +10,7 @@ meta_description: Manage access and set up permissions with user roles, organiza
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
```suggestion
<div class="enterprise-only"><p>
```

huguestennier commented on docs/source/guide/predictions.md:
```diff
@@ -823,7 +823,7 @@ If annotators can't see predictions or if you encounter unexpected behavior afte
 - Enable **Show predictions to annotators in the Label Stream and Quick View**
 - Select the relevant **Model Version** in the drop-down. If there is no drop-down menu visible, there might not be a model version listed for the pre-annotations, or there might be another issue happening. 
-- <i class='ent'></i> Disable the option to **Reveal pre-annotations interactively**, which requires manual action from annotators to display pre-annotated regions. (Label Studio Enterprise only)  
+- <div class="enterprise"><p> Disable the option to reveal pre-annotations interactively, which requires manual action from annotators to display pre-annotated regions.</p></div>
```
```suggestion
- <div class="enterprise-only"><p> Disable the option to reveal pre-annotations interactively, which requires manual action from annotators to display pre-annotated regions.</p></div>
```

huguestennier commented on docs/source/guide/quality.md:
```diff
@@ -10,9 +10,7 @@ meta_description: In data labeling projects, start evaluating annotator performa
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
```suggestion
<div class="enterprise-only"><p>
```

huguestennier commented on docs/source/guide/security.md:
```diff
@@ -11,9 +11,7 @@ Label Studio provides many ways to secure access to your data and your deploymen
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
```suggestion
<div class="enterprise-only"><p>
```

huguestennier commented on docs/source/guide/setup_project.md:
```diff
@@ -36,9 +36,7 @@ After you save a project, any other collaborator with access to the Label Studio
-<div class="admonition enterprise">
-    <p class="admonition-title">enterprise</p>
-    <p>
+<div class="enterprise"><p>
```
```suggestion
<div class="enterprise-only"><p>
```

huguestennier reviewed (changes requested):
I think all `<div class="enterprise">` should be changed to `<div class="enterprise-only">` right?

anithamuthu2022 commented:
@huguestennier could you help fix the merge conflict on docs/themes/v2/layout/partials/header.ejs?

anithamuthu2022 commented:
@huguestennier could you please review this commit to make sure everything looks good/requires a change? 
https://github.com/heartexlabs/label-studio/pull/3253/commits/f3785ddd759017302680b82b9d974c39b7be16ed

makseq commented on docs/source/guide/styleguide-enterprise.md:
```diff
@@ -0,0 +1,430 @@
+title: Styleguide for Enterprise
+short: Styleguide-Enterprise
+tier: enterprise
+type: guide
```
let's remove this line, this md file should be hidden in our docs

makseq replied:
the same for `styleguide.md`

anithamuthu2022 commented:
I've removed `type: guide` (line #4) from the following files so the testing pages are not visible on the left nav
another-subpage.md
test-sub-page.md

anithamuthu2022 commented:
This page `ts+video.md` is missing from the docs site. This also looks like a standalone page like the `pingone.md` file. 
https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/ts%2Bvideo.md
https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/pingone.md

Let me know if the `order` for these pages to get included in the docs website.

anithamuthu2022 commented:
`decommission_minio.md` is also a standalone page and is not visible on the docs website.

anithamuthu2022 commented:
@huguestennier @makseq I've addressed your feedback and also replaced the format for `Note` messages. 
When you get a chance please review this PR and we can merge it. Thank you!

makseq commented:
> This page `ts+video.md` is missing from the docs site. This also looks like a standalone page like the `pingone.md` file. https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/ts%2Bvideo.md https://github.com/heartexlabs/label-studio/blame/develop/docs/source/guide/pingone.md
> 
> Let me know the `order` for these pages to get included in the docs website.
> 
> The file naming format for `ts+video.md` is incorrect. I've renamed it as `timeseries_video.md`.

yes, it should be accessible on demand only
